my-app/src/history.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Popup from "reactjs-popup";
+
+
+class History extends React.Component{
+	render(){
+		return(
+			<div>
+				  <Popup trigger={<div className = "button" style = {{border: "solid", width: 45, height: 45}}><h2 style = {{font: 10, textAlign: 'center'}}>H</h2></div>} modal>
+				    
+				  	<div>
+				  		<h4>History</h4>
```

Comment:
For the components that dim the background and "pop up" over the main window, you need to have an "X" button that the user can click to close the pop up. It's not obvious that you need need to click outside the pop up to close it.